db/migrate/20180919032754_create_lesson_schedule.rb

```diff
@@ -0,0 +1,13 @@
+class CreateLessonSchedule < ActiveRecord::Migration[5.1]
+  def change
+    create_table :lesson_schedules do |t|
```

Comment:
Cool! Then you need to run rails db:migrate. If you are still having pg problems see this repo readme or tell me